.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "standard"
+}
```

Comment:
Not sure why this is needed, but if it's just for your local dev then I can't see any harm in it. Would be good to have a commit comment or something explaining why it's been added.

Reply:
That's right, I use the .eslint plugin in VSCode, and this ensures it uses the standard rules.  I added a commit comment when adding this file.